cmd/dogeboxd/server.go

```diff
@@ -98,6 +98,8 @@ func (t server) Start() {
 	wsh := dogeboxd.NewWSRelay(t.config, dbx.Changes)
 	rest := dogeboxd.RESTAPI(t.config, dbx, wsh)
 
+	dogeboxd.ServeUI(t.config)
```

Comment:
This will block the rest of the server startup, needs to be a Conductor service with a Run method

Reply:
✅ Fixed